src/GLFW.jl

```diff
@@ -22,8 +22,23 @@ else
 	error("GLFW $VERSION is not supported")
 end
 
+immutable GLFWError <: Exception
+    code::ErrorCode
```

Comment:
👍

If a GLFW (a newer version, for example) were to ever pass in an integer that isn't defined in the enum, what will happen when `GLFWError` is constructed?